docs/2101UserGuide.md

```diff
@@ -321,69 +324,104 @@ List of commands that can be undone:
 
 #### Redoing a command: `redo`
 
-Redoes a command that the user has undone previously.
+Accidentally used `undo`? Don’t worry, you can redo your undone commands to retrieve your previous data too!
 
-Format: `redo`
+Here’s how you can use it: `redo`
 
-<hr class="command-separator">
-
-#### Clear all Members and Events: `clear`
-Clears all Member and Event entries from CCACommander.
+Here's a list of commands you can redo:
+* `clear`
+* `createMember`
+* `deleteMember`
+* `editMember`
+* `createEvent`
+* `deleteEvent`
+* `editEvent`
+* `enrol`
+* `unenrol`
+* `editEnrolment`
 
-Format: `clear`
 
 <hr class="command-separator">
 
-#### Help: `help`
+#### Clear all Members and Events: `clear`
+Is it time for a new academic year when you need to remove all your previous members and events for new members and 
+events? You can use the `clear` command to instantly remove all your members and events without needing to delete them one by one.
 
-Displays a pop-out window that shows a link to this User Guide.
+Here’s how you can use it: `clear`
 
-Format: `help`
+<div markdown="block" class="alert alert-primary">:bulb: Accidentally cleared your list? 
+Worry not as you can use the `undo` command to retrieve your data! 
 
-<hr class="command-separator">
+</div>
 
-#### Exit: `exit`
+<hr class="command-separator">
 
-Closes CCACommander and its display window.
+#### Help: `help`
 
-Format: `exit`
+Forgot how to edit a specific enrolment or how to unenrol members? The `help` command will allow you to clarify and recall any features of the app by redirecting you to our friendly and informative user guide.
```

Comment:
I think not directly redirecting, so maybe
```suggestion
Forgot how to edit a specific enrolment or how to unenrol members? The `help` command will allow you to clarify and recall any features of the app by providing you with a redirection link to our friendly and informative user guide.
```

Reply:
I have changed it to `providing you with a link`